tools/test_env

```diff
@@ -84,6 +84,13 @@ elif [ "CHK$1" == "CHKmacvlan" ];then
     sudo ip link set link dev mac0 type macvlan macaddr add 00:22:22:22:22:22
     sudo ip link set eth1 up
     sudo ip link set mac0 up
+elif [ "CHK$1" == "CHKmacvtap" ];then
+    create_nics
+    sudo ip link add macvtap0 link eth1 type macvtap mode source
+    sudo ip link set link dev macvtap0 type macvtap macaddr add 00:11:11:11:11:11
+    sudo ip link set link dev macvtap0 type macvtap macaddr add 00:22:22:22:22:22
+    sudo ip link set eth1 up
+    sudo ip link set macvtap0 up
```

Comment:
You forgot the delete this interface in `rm` section.

Reply:
Oops. Fixed, thanks!